sdk/tests/deqp/functional/gles3/es3fShaderDerivateTests.js

```diff
@@ -284,6 +284,22 @@ goog.scope(function() {
         return value + numUlps * es3fShaderDerivateTests.getSingleULPForValue(value, numMantissaBits);
     };
 
+    /**
+     * @param {?gluShaderUtil.precision} precision
+     * @return {number}
+     */
+    es3fShaderDerivateTests.getInterpolationLostBitsWarning = function(precision) {
+        // number mantissa of bits allowed to be lost in varying interpolation
```

Comment:
Maybe an upstream typo? "number of mantissa bits"?

Reply:
Yes, this is an upstream typo.